src/rubintv/models/models_assignment.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import Any
+
+import yaml
+from dacite import from_dict
```

Comment:
This seems like quite a small/niche dependency to add for not a huge amount of functionality, which does give me some pause. How hard would it be to do what `from_dict` is doing without using this package?

Reply:
Not hard but possibly messy looking and therefore more open to bugs. It would be more transparent though. I'm happy to re-write without it if you think it'd be better.